core/src/avm2/globals/flash/events/AccelerometerEvent.as

```diff
@@ -0,0 +1,36 @@
+package flash.events
+{
+    
+    public class AccelerometerEvent extends Event
+    {
+        public static const UPDATE:String = "update"; // Defines the value of the type property of a AccelerometerEvent event object.
+
+        public var timestamp: Number; // The number of milliseconds at the time of the event since the runtime was initialized.
+        public var accelerationX: Number; // Acceleration along the x-axis, measured in Gs.
+        public var accelerationY: Number; // Acceleration along the y-axis, measured in Gs.
+        public var accelerationZ: Number; // Acceleration along the z-axis, measured in Gs.
```

Comment:
Have you checked the order of declaration for the fields - using `describeType()`?
Same question for all classes.

Reply:
I used the order of the inputs into the constructor with any additional properties not connected to an input from the constructor afterwards. That was my best guess based purely off the documentation when I wrote the generating code.

Reply:
I see. See the relevant conversation at: https://github.com/ruffle-rs/ruffle/pull/7558#pullrequestreview-1064281702
I did reorder the stuff in `Vector3D` accordingly. But again, this is not crucial, and as said above, in many cases we already have a mismatch with FP.